### DISCUSSION TITLE
APP_TOKEN可以更換嗎？

### DISCUSSION THREAD

**Discussion by nick3434** [osk2/panasonic_smart_app#55] [General]
請問每個登入的user會有自己的APP_TOKEN嗎(const.py)？如果有，換成自己所屬的APP_TOKEN能減少API過度使用的問題嗎？
謝謝！

路徑：panasonic_smart_app-master\custom_components\panasonic_smart_app\smartApp\const.py

**Comment by osk2** [comment]
沒有，`APP_TOKEN` 都是固定的

目前還沒有一個比較好的方法可以避開超量使用的問題

**Reply by xxzxzqs997** [reply]
明明输入的账号密码是对的，可是总是提示错误登录
![Uploading IMG_5342.png…]()
